### DISCUSSION TITLE
AWS-Marketplace Install, No alerts

### DISCUSSION THREAD

**Discussion by ob1xxx** [Security-Onion-Solutions/securityonion#7532] [Unsupported Versions] [answered]
Hello Friend,

So I've installed the latest build, 2.3.10, from the market place. I've enabled 2 network interfaces and have verified with traffic mirroring and tcpdump. So the SO instance does see the traffic and there are low level alerts. The issue is I'm actively attacking a victim box, but don't see any alerting. I'm using Metasploit on the victim machine and am popping shells like a mad man. Still no alerting. I've ran "so-allow" and selected "analyst". Any thoughts as to what is happening?

Traffic Mirroring- There are two sessions active. 
1. Victim box mirror to SO eth0(mgmt interface)
2. Attack(Metasploit) to SO eth0

**Comment by dougburks** [comment]
Are you saying that eth0 is your management interface?  

Are you sending the network traffic to your management interface or to your dedicated sniffing interface?

**Reply by ob1xxx** [reply]
Yes, eth0 is the management interface. Originally I was sending the traffic to mgmt interface. Then I decided to test by sending also sending traffic to the sniffing interface. Once I made this change I reran the attacks via metaslpoit, which was successful, but I don't see any alerts. I've attached a screenshot of the setup. 
![image](https://user-images.githubusercontent.com/20797451/158199090-5e476eac-553d-4810-98dc-b2eebf95a2d3.png)

**Reply by dougburks** [reply]
After sending traffic to the sniffing interface, do you see the attack traffic on that sniffing interface using tcpdump?

**Reply by ob1xxx** [reply]
Yes, I can see the attack traffic from TCPDUMP. This initially made me happy, then once I search alerts in SO, nada. Search for network connections in Kibana, nada. Search in Zeek, there's only one alert category, "notice". Strange.

**Reply by dougburks** [reply]
Are you getting more than just notice logs in `/nsm/zeek/logs/`?

When you configured the instance, what did you set your homenet to?
https://docs.securityonion.net/en/2.3/homenet.html

**Comment by ob1xxx** [comment]
Yes, just the notice logs under zeek. Currently, the homenet is set to a ip range at 172.31.0.0/12

**Comment by ob1xxx** [comment]
This is a standalone node BTW.

**Comment by ob1xxx** [comment]
In viewing the node configurations under minions, I'm seeing that both eth0 & eth1 have the same IP. When I configured the box this was done by selecting the image from AWS marketplace, launching it, then ssh to run ./sosetup-network. I've found instructions for setup in AWS at https://docs.securityonion.net/en/2.3/cloud-ami.html. I didn't use this. The results of ifconfig show two interfaces with only eth0 having an IP.

**Comment by dougburks** [comment, marked as answer]
If you didn't follow the instructions at https://docs.securityonion.net/en/2.3/cloud-ami.html, then it's probably best to start over with a new installation and follow the instructions this time.

**Comment by ob1xxx** [comment]
I went through the re-install with the instructions. Still the same issue. Not catching the alert traffic. Winlogbeats was added and works just fine. Being that this is stand alone the only section of setup instruction was

**Comment by dougburks** [comment]
Did you set your homenet to `172.31.0.0/12` again?

When you view the traffic using tcpdump, is that the IP range that is seen for your victim machine(s)?

Is Zeek logging as you would expect it to?

Have you verified that there are rules enabled to detect the specific attacks you are using?
